### ISSUE TITLE
nb_inspect_correlation_pnr does not work interactively

### ISSUE THREAD

**Issue by EricThomson** [flatironinstitute/CaImAn#760]
**tl;dr** 
When using `nb_inspect_correlation_pnr`, using the selection tool to adjust the range of y values does not change the images.

**Details**
1. Operating system (Linux/macOS/Windows): 
Windows 10
 
 2. Python version (3.x): 
 3.6.10

 3. Working environment (Python IDE/Jupyter Notebook/other):
Jupyter 

 4. Which of the demo scripts you're using for your analysis (if applicable):
demo_pipeline_cnmfE.ipynb
 
 5. CaImAn version*: 
1.8.3
 
 6. CaImAn installation process (`pip install .`/`pip install -e .`/conda): 
conda install caiman -c conda-forge

**Describe the issue that you are experiencing**
In `demo_pipeline_cnmfE.ipynb` I am running `nb_inspect_correlation_pnr`. The images and histograms show up fine, but when I use the selection tool to pick a range of y values, the images do not adjust depending on the range of y values selected.  

When I add a new cell and run the older function `inspect_correlation_pnr(cn_filter, pnr)` it works great, so that's what I'm doing. Maybe add this as a secondary option for people? If this seems reasonable I could make a PR adding this to the nb.

**Comment by epnev** [comment]
@EricThomson Thanks for making this note. We had a similar issue in the past. Does the suggestion [here](https://github.com/flatironinstitute/CaImAn/issues/744#issuecomment-624656711) solve the issue?

**Comment by EricThomson** [comment]
@epnev I am using holoviews v `1.13.2` and `hv.notebook_extension('bokeh')` is in the notebook that is showing the issue.

**Comment by epnev** [comment]
@j-friedrich @pgunn ^^ any ideas? I remember you looked at something similar recently.

**Comment by j-friedrich** [comment]
Using the selection tool on the correlation or pnr image does change the histograms, but not the other way around, though that would be useful.
Am not familiar with holoviews. Who wrote the holoviews code again? Maybe he knows a quick fix.

**Comment by pgunn** [comment]
I'm also not familiar with holoviews - I'll play around with it and see if I can figure out a fix.

**Comment by EricThomson** [comment]
> Using the selection tool on the correlation or pnr image does change the histograms, but not the other way around, though that would be useful.  Am not familiar with holoviews. Who wrote the holoviews code again? Maybe he knows a quick fix.

I believe it is @phildong (PR #610 ).

**Comment by epnev** [comment]
As an FYI the function works fine for me albeit in an older environment with holoviews 1.12.3

**Comment by pgunn** [comment]
This is very helpful - was having trouble reproducing the issue. Will try varying my holoviews version.

**Comment by EricThomson** [comment]
I downgraded to holoviews 1.12.3, got an import error with `hv.notebook_extension('bokeh')` -- `ImportError: None of the backends could be imported`. This is a known incompatibility between holoviews 1.12 and Bokeh 2+ (https://github.com/holoviz/holoviews/issues/4280). 

When I downgraded to Bokeh 1.4 *and* Holoviews 1.12, it seems things work, though maybe something to keep an eye on, compatibility-wise, moving forward?
